met-web/src/components/engagement/view/EngagementView.tsx

```diff
@@ -47,9 +48,12 @@ export const EngagementView = () => {
                     m={{ lg: '0 8em 1em 3em', md: '2em', xs: '1em' }}
                     rowSpacing={2}
                 >
-                    <Grid item xs={12} lg={8}>
```

Comment:
if both xs and lg are the same then there is no sense in having them both. xs tells the browser the width for screens that are x small and up, sm small and up and etc. If you specify xs={8} then it will tell it to be 8 for lg as well.

However, I do wish to consider mobile friendliness here, so test it on a small screen.

What I suggest is xs={12} for both content and widget and lg={8} for content and 4 for widget. Test it on different screen sizes

Reply:
it looks much better thank you